backend/app/routers/authorization.py

```diff
@@ -18,23 +23,62 @@ async def save_authorization(
     user=Depends(keycloak_auth.get_current_username),
     db: MongoClient = Depends(dependencies.get_db),
 ):
-    """Save authorization info in Mongo. This is a triple of dataset_id/user_id/role."""
-    authorization_db = AuthorizationDB(**authorization_in.dict(), creator=user)
+    """Save authorization info in Mongo. This is a triple of dataset_id/user_id/role/group_id."""
+
+    # Check all connection and abort if any one of them is not available
+    if db is None:
```

Comment:
We should start checking this in the dependencies themselves. Otherwise we have to add this code to every endpoint.

Reply:
will move this.